### DISCUSSION TITLE
mod_auth_openidc 2.4.11.1 - Windows 2016 - License

### DISCUSSION THREAD

**Discussion by GVijayAnand** [OpenIDC/mod_auth_openidc#947] [Q&A]
Dear All,

We are in the process of implementing mod_auth_openidc 2.4.11.1 module in Windows 2016.

During the loading process, below error message is reported:
[auth_openidc:error] [pid 7908:tid 100] oidc_config_verify_license: could not read license file: C:/Apache24/conf/mod_auth_openidc.lic

Could you please help/ provide pointers on how to acquire license - mod_auth_openidc.lic file/ limited trial version which we can use it for POC ?

Regards,
G.Vijay Anand

**Comment by zandbelt** [comment]
if you're a customer you can send your questions to support@zmartzone.eu, if. you're not you should discuss the trial via sales@zmartzone.eu

**Reply by GVijayAnand** [reply]
Thanks for your quick support and feedback. 

We will consult Support/ Sales team.

Regards,
G.Vijay Anand
